pages/checkout/terms.tsx

```diff
@@ -49,83 +61,132 @@ const InfoBox = styled.div`
   margin-bottom: ${tokens.spacings.comfortable.medium}
 `
 
+const DataSourceErrorContainer = styled.div`
+  > *:not(:last-child) {
+    margin-bottom: ${tokens.spacings.comfortable.medium};
```

Comment:
Side note: We might have reached the point where we could make life a little bit easer for ourselves if we added some css variables for the spacing tokens
`--spacing-medium: tokens.spacings.comfortable.medium`, `--spacing-md: tokens.spacings.comfortable.medium` `--spacing-8: tokens.spacings.comfortable.medium` or whatever floats your boat

Reply:
they'd be added in `global.css`, right?

Reply:
Yes